src/Problem/Serializer/ErrorNormalizer.php

```diff
@@ -35,7 +37,7 @@ final class ErrorNormalizer implements NormalizerInterface, CacheableSupportsMet
         self::TITLE => 'An error occurred',
     ];
 
-    public function __construct(private readonly bool $debug = false, array $defaultContext = [])
+    public function __construct(private readonly IriConverterInterface $iriConverter, private readonly bool $debug = false, array $defaultContext = [])
```

Comment:
you can't break this class, the argument must be last position and nullable + add an expect deprecation although I'm quite sure we can handle it with a default IriConverter to avoid the deprecation alltogether.